lib/src/tasks/test/cli.dart

```diff
@@ -85,6 +89,8 @@ class TestCli extends TaskCli {
 
     bool unit = parsedArgs['unit'];
     bool integration = parsedArgs['integration'];
+    bool testNamed;
+    parsedArgs['name'] != null ? testNamed = true : '';
```

Comment:
This syntax is a bit weird. Ternary is usually used like so:
```dart
bool testNamed = parsedArgs['name'] != null ? [value if true] : [value if false]
```
In this case, it's just a bool, so it could be:
```dart
bool testNamed = parsedArgs['name'] != null;
```
Another thing to note - as it is currently written, `testNamed` will either be `true` or `null`, which makes me think the if statement below on line 167 could throw (null boolean).